notebooks/Notebook2.md

```diff
@@ -9,6 +9,11 @@ Did it help you make a design decision? Was it helpful in addressing the most pr
 issues in your project? How will you incorporate the feedback into your work? Will you
 change something about the design, implementation, or evaluation as a result?
 
+Olivia mentioned that the syntax for splitting into two sections seemed a bit too verbose.
+I agree with this, but am at a slight loss in how to make it more concise. Currently, there 
+are 3 parts to it: First is the parts definition (where you say how many parts there will be and
+define their names. For example, A&B). Then, two more parts where each represents the different part (one would represent A, the other B). I'm not sure how I could make it more concise.
```

Comment:
It's also okay if you don't know how to make it more concise right now. Even when we know that our language should have a particular property, we don't always know how to give it that property. Sometimes, we have to implement the whole thing, which can then give us the perspective we need. Then, we can revise the design. Knowing what should go in "version 2" can be an important part of the project!